README.md

```diff
@@ -22,10 +22,11 @@ Must contain the keys:
 - `script`: script with model definition.
   - `source`: relative path from config file to the script
   - `hash`: hash of the script file
+  - `name`: Name of class defining the model in `script`
```

Comment:
maybe keep `class_name`, to not confuse this key with the name of the script (file name).

Reply:
Hmm, I preferred the symmetry with the other options, now it's `name` and `kwargs` everywhere.

Reply:
yeah, nice solution with `definition`